src/PKSim.Core/Services/MoleculeOntogenyVariabilityUpdater.cs

```diff
@@ -55,8 +54,14 @@ public void UpdatePlasmaProteinsOntogenyFor(Population population)
       public void UpdateMoleculeOntogeny(IndividualMolecule molecule, Ontogeny ontogeny, Individual individual)
       {
          molecule.Ontogeny = ontogeny;
-         molecule.OntogenyFactorGI = _ontogenyRepository.OntogenyFactorFor(ontogeny, CoreConstants.Groups.ONTOGENY_DUODENUM, individual.OriginData);
-         molecule.OntogenyFactor = _ontogenyRepository.OntogenyFactorFor(ontogeny, CoreConstants.Groups.ONTOGENY_LIVER, individual.OriginData);
+         updateOntogenyParameter(molecule.OntogenyFactorGIParameter, _ontogenyRepository.OntogenyFactorFor(ontogeny, CoreConstants.Groups.ONTOGENY_DUODENUM, individual.OriginData));
+         updateOntogenyParameter(molecule.OntogenyFactorParameter, _ontogenyRepository.OntogenyFactorFor(ontogeny, CoreConstants.Groups.ONTOGENY_LIVER, individual.OriginData));
+      }
+
+      private void updateOntogenyParameter(IParameter parameter, double value)
+      {
+         parameter.DefaultValue = value;
```

Comment:
When an ontogeny parameter is updated, we set the VALUE And the default value. It was done for AGP and ALB but not for OF and OF GI